charts/argo-templates/scripts/diff-bq-table.sh

```diff
@@ -15,11 +15,11 @@ for c in ${COMPARE_COLS//,/ }; do
 done
```

Comment:
Thoughts for after I'm gone: The logic in this script & the others that interact with BQ might be complex enough now to justify spinning out a proper CLI, published as a Docker tool. It'd make testing (automated and manual) much easier.

Reply:
Good idea, want me to make a ticket and throw it in the backlog?

Reply:
Makes sense to me!